### ISSUE TITLE
Logout button not working

### ISSUE THREAD

**Issue by Winna95** [NoroffFEU/agency.noroff.dev#549]
See: https://github.com/NoroffFEU/agency.noroff.dev/pull/544
After this change, the content of navbar change according to the logged in user's role. I notice when clicking the logout button in the navbar nothing happens: The navbar content remain unchanged, it does not seem the values persisted to the storage solution upon succesful login are removed/reset. It looks like the user is still logged in after clicking the logout button. 

see login.js#login (currently at L63 + L67-71) for the values set in the storage solution upon successful login. Some of these might need to be removed/reset upon logging out. 

ui/header/index.js #checkState should probably somehow be re-run after cleaning up the storage.

**Comment by nyolarraklay** [comment]
hi, what is the status of this issue?

**Comment by Winna95** [comment]
@nyolarraklay still working on it, will finish it today

**Comment by bLangved** [comment]
@Winna95 This is probably a part of #672 
Instead of having 10 people work on small sections of the overall header-code, it looks beneficial to take the whole header as an issue. There will otherwise be a large number of merge conflicts, since people are reworking each others code all the time. 

Mind contacting me on either discord (langved) or teams, and we can implement the code you have written?

**Comment by Winna95** [comment]
@bLangved I already made a pull request on this. I thought the task was to make small issues to work on this week. However I did not get any merg-conflicts when pushing my branch. 
 This could solved as part of what you mentioned and it could be a smaller issue, depends on how granulated the tasks are. Merge conflicts may occur from time to time, when we are working in large teams. The impact of these however are typically mitigated when keeping the amount of changes in each pull request fairly small.
